package.json

```diff
@@ -22,19 +22,23 @@
     "raw-loader": "^0.5.1",
     "to-string-loader": "^1.1.4",
     "typescript": "~1.8.9",
-    "typings": "~1.0.3",
+    "typings": "~1.3.2",
     "webpack": "^1.12.9",
     "webpack-dev-server": "^1.14.0",
-    "webpack-merge": "^0.8.4"
+    "webpack-merge": "^0.14.1"
   },
   "dependencies": {
-    "@angular/common": "2.0.0-rc.4",
-    "@angular/compiler": "2.0.0-rc.4",
-    "@angular/core": "2.0.0-rc.4",
-    "@angular/http": "2.0.0-rc.4",
-    "@angular/platform-browser": "2.0.0-rc.4",
-    "@angular/platform-browser-dynamic": "2.0.0-rc.4",
-    "@angular/router": "3.0.0-beta.2",
+    "@angular/common": "2.0.0-rc.5",
+    "@angular/compiler": "2.0.0-rc.5",
+    "@angular/core": "2.0.0-rc.5",
+    "@angular/http": "2.0.0-rc.5",
+    "@angular/compiler-cli": "0.5.0",
+    "@angular/platform-server": "2.0.0-rc.5",
+    "@angular/forms": "0.3.0",
+    "@angular/platform-browser": "2.0.0-rc.5",
+    "typescript": "^1.9.0-dev",
```

Comment:
Keep forms if we are using them. Let's not go into AoT in this PR. Let's focus on a minimal set of changes to get RC5 compatibility. Baby steps :-)